### ISSUE TITLE
Can you explain the impact of service worker?

### ISSUE THREAD

**Issue by karkovich** [cwilso/metronome#27]
I've just read your article https://web.dev/audio-scheduling/ and it seems like you said that the combination of
setTimeout/Interval with AudioContext make the ```Rock-Solid Timing```.
But in the example it is a bit more complex than the article.

1) Is any option that you explain more about the impact of service workers?

**Comment by cwilso** [comment]
I think you mean the worker, not service worker?  I don't think I ever set up a SW for this "app".

Unfortunately, the migration from HTML5Rocks.com to web.dev lost the comment stream where I had explained this.  This was added because Chrome (and other browsers, I think) made a power-saving change to throttle JS timers down.  I believe this is actually no longer necessary, because based on ANOTHER change over a year ago (Jake wrote an article outline the current state of affairs at https://developer.chrome.com/blog/timer-throttling-in-chrome-88/) audio will prevent the throttling.

Timers in worker threads were not throttled when I wrote this code, so I set up a worker thread to ping the main thread (which does audio scheduling).   I think this is no longer necessary, and setTimeout on the main thread would work just fine.
